src/components/reusable/notification/notification.scss

```diff
@@ -5,9 +5,9 @@
 :host {
   display: block;
   width: 100%;
-  kd-card {
+  kyn-card {
     width: 100%;
-    margin-top: 8px;
+    padding-bottom: 8px;
```

Comment:
Don't put padding/margin here. Use CSS in the notifcation-panel component to create the gaps.

Reply:
This padding is still here.

Reply:
oh ok removing this

Reply:
here you go. Its done and removed from here.